BATTLESHIP/readme.md

```diff
@@ -0,0 +1,101 @@
+# The Applications
+This application is a Battleships game.
+
+The game supports two players playing against each other.
+
+The game follows the standard rules - 10x10 grid, 5 ships, single shot per turn.
```

Comment:
Can we add a link to purchase the e-paper display? Bruno will post it in WhatsApp

Reply:
that is in the markdown of the top page..